tests/pipeline/default/sgdemux.nf.test

```diff
@@ -0,0 +1,30 @@
+nextflow_pipeline {
+
+    name "Test SGDEMUX"
+    script "main.nf"
+
+    test("Should run without failures") {
+
+        when {
+            params {
+                input = "$baseDir/assets/inputs/sgdemux-samplesheet.csv"
+                demultiplexer = 'sgdemux'
+                outdir = "$outputDir"
+            }
+        }
+
+        then {
+            assert workflow.success
+            assert workflow.trace.succeeded().size() == 104
+
+            assert snapshot(
```

Comment:
fastqs

Reply:
cannot assert md5sums due to sorting, but I have added asserts that the fastqs exist 125fff34b177f5af359fff6e5b6d5fd227bf1800